src/components/News.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import { useReducer } from "react";
+import useNews from "../hooks/useNews";
+
+function News(): JSX.Element {
+  const { data } = useNews();
+  return (
+    <>
+      <h1>Hello World</h1>
+      {data.map((d, i) => {
```

Comment:
You should never use the index as a key, it's an invalid key that doesn't work as expected - it will not track the right item on change.

Reply:
Yes. it is an anti pattern. I used this here for simplicity. Once you have your data you can use ids of your items.